addon/helpers/markdown-viewer.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+
+const { htmlSafe } = Ember.String;
+
+/**
+Render markdown text for selected rules.
+
+ Usage example:
+ ```handlebars
+ This is the text we want to render: {{markdown-viewer '*text is italic*'}}
+ ```
+
+ @class markdown-viewer-helper
+ @uses markdownit
+ */
+/* global markdownit */
+export function markdownViewer(params) {
+
+    const [markdown] = params;
+    const html = markdownit('zero', {
```

Comment:
Looks like you're using "zero" to disable all rules, and then enable only a select few. If so, are L21-37 doing anything? If not, they can be removed.